### PR TITLE
utils/gnupg: Add build of gpgv

### PR DESCRIPTION
**Pull request by cshoredaniel**
Maintainer: @psycho-nico 
Compile tested: x86_64, LEDE trunk
Run tested: x86_64, LEDE trunk, debootstrap of ubuntu with ubuntu keyring (which uses gpgv)

Description:

gpgv is a stripped down gnupg useful for only verifying signatures.
Having this package can save space when all you need is signature
verification.

Signed-off-by: Daniel Dickinson lede@cshore.thecshore.com

### CODE REVIEW

**Comment by yousong** [comment]
@openwrt/package-maintainers this pr has been sitting here for more than one month now without receiving any comments from the current maintainer, maybe we should do a maintainership transfer or collect severals approvals/disapprovals to do the decision in the fast way.

**Comment by diizzyy** [comment]
@hnyman 
Can we merge this?

**Comment by hnyman** [comment]
@cshoredaniel 
Want to take over maintainership? 

I think that @psycho-nico is MIA as he has not responded to any PRTs for ages. (his last commit here is from Feb2016, I think)

**Comment by cshoredaniel** [comment]
I'll do an update with myself a maintainer in the next few days.

**Comment by cshoredaniel** [comment]
Added a commit with myself as maintainer and a version bump

File: utils/gnupg/Makefile [hunk 1]
```diff
@@ -37,6 +38,12 @@ define Package/gnupg
   MENU:=1
 endef
 
+define Package/gpgv
+  $(call Package/gnupg/Default)
+  TITLE+= (signature verification only)
+  MENU:=1
```
Comment (yousong) [inline utils/gnupg/Makefile]: There should be no need for the `MENU:=1` thing?

Comment (cshoredaniel) [inline utils/gnupg/Makefile]: Ok, fixed.  I've also shortened the gpgv package title so it's more likely visible in menuconfig than the old one.

**Comment by yousong** [comment]
On 12 January 2017 at 13:34, Daniel Dickinson <notifications@github.com>
wrote:

> *@cshoredaniel* commented on this pull request.
> ------------------------------
>
> In utils/gnupg/Makefile <https://github.com/openwrt/packages/pull/3261>:
>
> > @@ -37,6 +38,12 @@ define Package/gnupg
>    MENU:=1
>  endef
>
> +define Package/gpgv
> +  $(call Package/gnupg/Default)
> +  TITLE+= (signature verification only)
> +  MENU:=1
>
> Ok, fixed. I've also shortened the gpgv package title so it's more likely
> visible in menuconfig than the old one.
>

​Not sure what happened, but github now gives me 404 error after clicking
"view it on GitHub"​ and i also cannot find the pr on the project page...

**Comment by cshoredaniel** [comment]
@yousong I'm not sure what is up - I think github had a fit because I did keystrokes thinking I was in a different window that did some nasty thing that requires manual intervention from github.  anyway this commit is back now.

**Pull request merged**